src/adlmidi_load.cpp

```diff
@@ -72,59 +72,50 @@ bool MIDIplay::LoadBank(const void *data, size_t size)
     return LoadBank(file);
 }
 
-
-struct WOPL_Inst
-{
-    bool fourOps;
-    char padding[7];
-    struct adlinsdata adlins;
-    struct adldata    op[2];
-    uint16_t ms_sound_kon;
-    uint16_t ms_sound_koff;
-};
-
```

Comment:
Yeah, it was just a convenient proxy, and it's good when you have made direct passing of the data into final store :+1: